core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/offer/domain/FulfillmentGroupAdjustment.java

```diff
@@ -29,5 +29,26 @@
     public void setValue(Money value);
 
     public void setFulfillmentGroup(FulfillmentGroup fulfillmentGroup);
-    
+
+    /**
+     * Future credit means that the associated adjustment will be discounted at a later time to the customer 
+     * via a credit. It is up to the implementor to decide how to achieve this. This field is used to determine 
+     * if the adjustment originated from an offer marked as FUTURE_CREDIT.
+     *
+     * See {@link Offer#getAdjustmentType()} for more info
+     * 
+     * @return
+     */
+    Boolean isFutureCredit();
```

Comment:
Would have liked this to be an enum for adjustment type but ok as is for this release.